content/en/docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -971,7 +971,8 @@ Keep watching the `StatefulSet`'s Pods in the first terminal and drain the node
 `zk-1` is scheduled.
 
 ```shell
-kubectl drain $(kubectl get pod zk-1 --template {{.spec.nodeName}}) --ignore-daemonsets --force --delete-emptydir-data "kubernetes-node-ixsl" cordoned
+kubectl drain $(kubectl get pod zk-1 --template {{.spec.nodeName}}) --ignore-daemonsets --force --delete-emptydir-data
+"kubernetes-node-ixsl" cordoned
```

Comment:
How about this? 😊
(Separating input and output)

```suggestion
kubectl drain $(kubectl get pod zk-1 --template {{.spec.nodeName}}) --ignore-daemonsets --force --delete-emptydir-data
# Put ``` here!
# Put an empty line here!
# Put ```output here!
"kubernetes-node-ixsl" cordoned
```

Reply:
I'm fine with that!👍